### PR TITLE
Angular material pagination added

### PR DESCRIPTION
**Pull request by shubhwale**
Angular material table and pagination added

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
     "bootstrap": "^4.4.0",
     "first": "0.0.3",
     "fundamental-ngx": "0.6.0",
+    "ngx-pagination": "^5.0.0",
```
Comment (ketannaik) [inline package.json:29]: You can remove ngx-pagination library from the code

File: src/app/app.module.ts [hunk 2]
```diff
@@ -20,6 +20,10 @@ import { EditBookComponent } from './books/edit-book/edit-book.component';
 import { CartComponent } from './cart/cart.component';
 import { BookService } from './shared/book.service';
 import { MatBadgeModule } from '@angular/material/badge';
+import { NgxPaginationModule } from 'ngx-pagination';
```
Comment (ketannaik) [inline src/app/app.module.ts:23]: Same here

File: package.json (hunk 1 above)
Comment (shubhwale) [reply package.json]: Done

File: src/app/app.module.ts (hunk 2 above)
Comment (shubhwale) [reply src/app/app.module.ts:23]: Done

**Pull request merged**